cognite/neat/_rules/models/dms/_validation.py

```diff
@@ -376,6 +400,7 @@ def _validate_reverse_connections(
         self,
         properties_by_ids: dict[tuple[dm.ViewId, str], ViewPropertyApply | ViewProperty],
```

Comment:
naming ! properties_by_view or simply view_properties as you are not getting properties dict where key is property id but key is view id